src/Shared/Controller/HealthcheckController.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Shared\Controller;
```

Comment:
Outside of the scope of this PR: we need to add the MIT license headers and the LICENSE.md